packages/plugins/src/plugins/sushiswap/helpers.ts

```diff
@@ -50,3 +58,134 @@ export async function getV3PairsAddresses(url: string, length = 300) {
 
   return pools.map((p) => sushiV3PairToUniV2(p));
 }
+
+export async function getSushiswapTokens(
+  networkId: EvmNetworkIdType
+): Promise<Address[]> {
+  const response = await fetch(`${SUSHISWAP_TOKEN_LIST_URL}${networkId}.json`);
+  if (!response.ok) {
+    throw new Error(`Failed to fetch token list: ${response.statusText}`);
+  }
+
+  const tokenList = await response.json();
+  return tokenList.map(
+    (token: { address: string }) => token.address.toLowerCase() as Address
+  );
+}
+
+export async function getV1Pools(
+  client: PublicClient,
+  masterChefAddress: Address
+): Promise<FarmingPool[]> {
+  const poolLength = await client.readContract({
+    address: masterChefAddress,
+    abi: farmingAbiV1,
+    functionName: 'poolLength',
+  });
+
+  const indices = Array.from({ length: Number(poolLength) }, (_, i) => i);
+
+  // For V1, we get LP tokens from poolInfo
+  const poolInfosRes = await client.multicall({
+    contracts: indices.map(
+      (idx) =>
+        ({
+          address: masterChefAddress,
+          abi: farmingAbiV1,
+          functionName: 'poolInfo',
+          args: [BigInt(idx)],
+        } as const)
```

Comment:
```suggestion
        } )
```

99.9% sure this const doesnt do anything if the abi is a const